limesurvey/limesurvey.py

```diff
@@ -91,6 +103,18 @@ def studio_submit(self, data, suffix=""):  # pylint: disable=unused-argument
 
         return {"result": "success"}
 
+    @XBlock.json_handler
+    def get_survey_url(self, data, suffix=''): # pylint: disable=unused-argument
+        """
+        Show the survey URL and access code to the user
+        """
+        URL = "http://limesurvey.local.overhang.io:8082"
```

Comment:
This shouldn't be hardcoded, should we add a patch in the tutor-contrib-limesurvey plugin that adds the LIMESURVEY_HOST to the LMS settings?

Reply:
For example: 
For the dev env: https://github.com/overhangio/tutor-notes/blob/master/tutornotes/patches/openedx-lms-development-settings#L1
For the production env: https://github.com/overhangio/tutor-notes/blob/master/tutornotes/patches/openedx-lms-production-settings

Reply:
Done